freqtrade/commands/analyze_commands.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from pathlib import Path
+from typing import Any, Dict
+
+from freqtrade.configuration import setup_utils_configuration
+from freqtrade.enums import RunMode
+from freqtrade.exceptions import OperationalException
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup_analyze_configuration(args: Dict[str, Any], method: RunMode) -> Dict[str, Any]:
+    """
+    Prepare the configuration for the entry/exit reason analysis module
+    :param args: Cli args from Arguments()
+    :param method: Bot running mode
+    :return: Configuration
+    """
+    config = setup_utils_configuration(args, method)
+
+    no_unlimited_runmodes = {
+        RunMode.BACKTEST: 'backtesting',
+    }
+    if method in no_unlimited_runmodes.keys():
+        from freqtrade.data.btanalysis import get_latest_backtest_filename
+
+        btfile = Path(get_latest_backtest_filename(config['user_data_dir'] / 'backtest_results'))
```

Comment:
Looking at this now - i think we may wan't to use `exportfilename` (from the config object) instead, and add `exportfilename` to the arguments (so you get `--backtest-filename <asdf>`)

that'll allow to specify explicit backtesting files (also older ones) - which is currently impossible, and will remove the need to have `backtest_results` distributed all around.

Please be aware that 'backtest_results' can be a directory (default) - or a file (if provided explicitly) - so that'll need separate handling.

Reply:
Hmm would there be a problem resolving the strategy name from the backtest_results metadata if people supply their own exportfilename?

Reply:
it's always alongside that.
on very old versions (prior to the meta file) that may not be the case - but i don't think THAT old versions can still be shown correctly anyway

Reply:
ah OK, so:

`freqtrade\commands\analyze_commands.py` `process_entry_exit_reasons(config['exportfilename'], ...`

And that's literally it? get_backtest_stats() already checks whether its a dir/file....